sdk/appconfiguration/app-configuration/test/internal/helpers.spec.ts

```diff
@@ -136,7 +136,9 @@ describe("helper methods", () => {
         url: "unused",
         abortSignal: {
           aborted: true,
+          // eslint-disable-next-line @typescript-eslint/no-empty-function
           addEventListener: () => {},
```

Comment:
I think this would satisfy the rule.
```suggestion
          addEventListener: () => { /* empty function */ },
```